### PR TITLE
Youngsville PD - PPRR 2017-2019

### PR DESCRIPTION
**Pull request by pckhoi**

### CODE REVIEW

File: clean/youngsville_pd.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+def clean_pprr_19():
+    return pd.read_csv(data_file_path(
+        'raw/youngsville_pd/youngsville_csd_pprr_2019.csv'
+    )).pipe(clean_column_names)\
+        .drop(columns=['department', 'department_name_category', 'type'])\
+        .pipe(set_values, {
+            'agency': 'Youngsville PD',
+            'salary_freq': salary.YEARLY,
+            'salary_year': 2019,
+            'employment_status': 'full-time'
```
Comment (pckhoi) [inline clean/youngsville_pd.py:46]: @ayyubibrahimi "Fulltime" text only appear in 2019 data so I only add it here.

File: match/youngsville_pd.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+    matcher = ThresholdMatcher(ColumnsIndex(["fc"]), {
+        "last_name": JaroWinklerSimilarity(),
+        "first_name": JaroWinklerSimilarity(),
+    }, dfa, dfb)
+    decision = 0.9
+    matcher.save_pairs_to_excel(data_file_path(
+        "match/youngsville_pd_pprr_2017_2019_v_post_pprr_2020_11_06.xlsx"), decision)
+    matches = matcher.get_index_pairs_within_thresholds(lower_bound=decision)
+    return extract_events_from_post(post, matches, 'Youngsville PD')
```
Comment (pckhoi) [inline match/youngsville_pd.py:69]: Here is where the POST events are extracted.

File: fuse/youngsville_pd.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    pprr = pd.read_csv(data_file_path(
+        'match/pprr_youngsville_pd_2017_2019.csv'
+    ))
+    post_event = pd.read_csv(data_file_path(
+        'match/post_event_youngsville_pd_2020.csv'
+    ))
+    events_df = rearrange_event_columns(pd.concat([
+        fuse_events(pprr),
+        post_event,
+    ]))
```
Comment (pckhoi) [inline fuse/youngsville_pd.py:41]: And here is where the POST events are fused with other events.

Comment (ayyubibrahimi) [reply fuse/youngsville_pd.py:41]: Yeah. I was being an idiot. Please disregard.

**Review by ayyubibrahimi — APPROVED** [review]

**Pull request merged**